cmd/agent/main.go

```diff
@@ -28,13 +28,12 @@ var (
 	Date    string
 )
 
+var debug = pflag.BoolP("debug", "d", false, "enable debug logging")
```

Comment:
```suggestion
var debug = pflag.BoolP("debug", "v", false, "enable verbose logging")
```

Optional, most if not all tools I am using have a -v flag for verbose logging; might make sense to change this here as well